### PR TITLE
About

### LINKED ISSUE #564: Add about page
Here's the mock ([link to the specs](https://www.figma.com/file/dUL9ybnag2LhQpicJseFaH/Seismic-Risk?node-id=1643%3A0)): 
About page

- the TFSG box will display the elements on top of each other on mobile
- the bottom logo row will also display the elements on top of each other on mobile

Here's the MKBT logo that leads to mkbt.ro:

### PR DESCRIPTION
**Pull request by CatalinCsnMaster**
### What does it fix?

Closes #564

### CODE REVIEW

File: client/src/containers/about/AuthorsFragment.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        <Paragraph style={{ padding: '1.5rem' }}>
+          <Trans>Proiect realizat de</Trans>
+        </Paragraph>
+        <img src={MAME1} alt="Make it better" width="186.36px" height="80px" />
+      </Col>
+      <Col sm={20} md={12} className="about-authors-col">
+        <Paragraph style={{ padding: '1.5rem' }}>
+          <Trans>Proiectat de</Trans>
+        </Paragraph>
+        <img src={CfR} alt="Code for Romania" width="186.36px" height="80px" />
```
Comment (Utwo) [inline client/src/containers/about/AuthorsFragment.js:23]: You don't need to specify px unit in width and height. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-width

**Review by Utwo — COMMENTED** [review]
Nice! Thank you! :fire:

File: client/src/containers/about/AuthorsFragment.js (hunk 1 above)
Comment (Utwo) [reply client/src/containers/about/AuthorsFragment.js:23]: Also, the content is not aligned properly to the left of the page. Tested in Firefox 
image

Comment (CatalinCsnMaster) [reply client/src/containers/about/AuthorsFragment.js:23]: In Safari and Chrome is aligned perfectly. I will install Firefox to see what is the problem.

Comment (CatalinCsnMaster) [reply client/src/containers/about/AuthorsFragment.js:23]: Screen Shot 2021-03-03 at 8 07 19 PM

Firefox on MacOs Catalina. I can't fix a problem that I can't see. Can you please double check?

Comment (Utwo) [reply client/src/containers/about/AuthorsFragment.js]: Maybe is from my Linux machine. Alright then, we can merge this and if the problem persist on my side, I will open a new issue.

**Review by Utwo — APPROVED** [review]
:fire:

**Review by tudoramariei — APPROVED** [review]

**Review by Utwo — APPROVED** [review]

**Pull request merged**